infrastructure/cli/pkg/cmd/create/create.go

```diff
@@ -48,14 +49,45 @@ func create(cmd *cobra.Command, args []string) {
 		fmt.Println("setting up Helm failed with err: ", err)
 		os.Exit(1)
 	}
-	if err := helm.InstallCharts(); err != nil {
+	if err := helm.InstallCharts(provider.GetHelmOverrides()); err != nil {
 		fmt.Println("installing Helm charts failed with err: ", err)
 		os.Exit(1)
 	}
 
+	if err = context.Store(); err != nil {
+		fmt.Println("could not store the kube context: ", err)
+		os.Exit(1)
```

Comment:
probably time to introduce a Exit() method that does `fmt.Println("❌ " + msg); os.Exit(1)`

I think "soon enough" we'll do a small internal `console` package to embed the knowledge/convention of how we do output but for now I'd add the function here